docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -4,31 +4,44 @@
 <titleabbrev>Nodes reload secure settings</titleabbrev>
 ++++
 
-
 The cluster nodes reload secure settings API is used to re-load the keystore on each node.
 
-[source,console]
---------------------------------------------------
-POST _nodes/reload_secure_settings
-POST _nodes/nodeId1,nodeId2/reload_secure_settings
---------------------------------------------------
-// TEST[setup:node]
-// TEST[s/nodeId1,nodeId2/*/]
+[[cluster-nodes-reload-secure-settings-api-request]]
+==== {api-request-title}
+`POST _nodes/reload_secure_settings` +
+
+`POST _nodes/nodeId1,nodeId2/reload_secure_settings` +
+
+`POST _nodes/_local/reload_secure_settings`
+
+[[cluster-nodes-reload-secure-settings-api-desc]]
+==== {api-description-title}
+Secure settings are stored in an on-disk keystore. When you have updated
+the secure settings in your keystore, you can use this API to reload those
```

Comment:
mention that this applies to some of these, which are marked as 'reloadable` in relevant documentation ?

Reply:
Ah, good reminder. Thanks.